replit_river/client.py

```diff
@@ -15,14 +15,19 @@
 logger = logging.getLogger(__name__)
 
 
-class Client:
+A = TypeVar("A")
```

Comment:
is it normal to have this be a one-letter generic? i would prefer something with more detail (e.g. `HandshakeMeta`) but in the case that this is just convention, a comment here would be nice

Reply:
It's not super common since having a real name for a generic can often get confused with a class name or a real definition in some capacity. I'm sympathetic though, I'll give it a shot.

Reply:
Settled on `HandshakeType`, as to avoid any sort of overlap with real classes.